packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-basic-sign-in/index.md

```diff
@@ -22,7 +22,7 @@ This use case requires the **password** factor.
 
 ## Configuration Updates
 
-If not already done, complete the steps that are described in [download and set up the SDK, widget, and sample app](/docs/guides/oie-embedded-common-download-setup-app/aspnet/main/). The steps guide you to the appropriate repository and location of the embedded Sign-In Widget sample app, identify the packages to install, and describe additional changes to the Okta org that are required for the widget app.
+Ensure that you've completed the steps to [download and configure the SDK, Sign-In Widget, and sample app](/docs/guides/oie-embedded-common-download-setup-app/aspnet/main/). These steps guide you to the appropriate repository and location of the embedded Sign-In Widget sample app, identify the packages to install, and describe additional changes to the Okta org that are required for the Widget.
```

Comment:
I think there are lots of instances of "If not already done", not just this one. Maybe see about changing them all to what you have here "Ensure that you've completed..."

Reply:
Thanks. I'm going to update all the "Configuration updates" for the Widget use cases since it's kind of in the same scope of this ticket. There are plans to clean up the other embedded SDK use cases later on.